app/lib/getOpeningTimesMessage.js

```diff
@@ -0,0 +1,38 @@
+function getOpeningHoursMessage(status) {
+  if (status.isOpen) {
+    const closedNext = status.until;
```

Comment:
I've suggested `until` is renamed as I found it a bit confusing. I think something like `nextStateChange` would be more appropriate as it would mean `status.until` can be assigned to the variables as it is here and it wouldn't seem as odd.